src/test/java/io/jenkins/plugins/analysis/core/views/DetailFactoryTest.java

```diff
@@ -0,0 +1,451 @@
+package io.jenkins.plugins.analysis.core.views;
+
+import edu.hm.hafner.analysis.Issue;
+import edu.hm.hafner.analysis.Issues;
+import edu.hm.hafner.analysis.Priority;
+import hudson.model.Job;
+import hudson.model.Run;
+import io.jenkins.plugins.analysis.core.model.AnalysisResult;
+import io.jenkins.plugins.analysis.core.model.StaticAnalysisLabelProvider;
+import org.eclipse.collections.impl.factory.Lists;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.util.Arrays;
+import java.util.List;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+/**
+ * Tests the class {@link DetailFactory}.
+ *
+ * @author Manuel Hampp
+ */
+class DetailFactoryTest {
+
+    /**
+     * Checks that a link to the fixed view, returns a FixedWarningsDetail-View
+     */
+    @Test
+    void shouldReturnFixedWarningsDetailWhenCallWithFixedLink() {
+        DetailFactory detailFactory = new DetailFactory();
+        Run<?, ?> owner = mock(Run.class);
+        AnalysisResult result = mock(AnalysisResult.class);
+        Issues<?> allIssues = mock(Issues.class);
+        Issues<?> newIssues = mock(Issues.class);
+        Issues<?> outstandingIssues = mock(Issues.class);
+        Issues<?> fixedIssues = mock(Issues.class);
+        Charset sourceEncoding = Charset.defaultCharset();
+        IssuesDetail parent = mock(IssuesDetail.class);
+
+        Object fixedWarningsDetail = detailFactory.createTrendDetails("fixed", owner, result, allIssues, newIssues, outstandingIssues, fixedIssues, sourceEncoding, parent);
+
+        assertThat(fixedWarningsDetail).isInstanceOf(FixedWarningsDetail.class);
+    }
```

Comment:
Zusätzlich wäre noch schön, wenn hier tatsächlich in Fixed ein Issue wäre und dann:
```
((FixedWarningsDetail)detail).getIssues().isEqualTo(fixedIssues);
```
Bzw. alternativ bei Verwendung von Stubs:
```
((FixedWarningsDetail)detail).getIssues().isSameAs(fixedIssues);
```